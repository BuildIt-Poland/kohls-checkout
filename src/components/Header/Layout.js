import styled from 'styled-components';

import { colorLightGrey, borderWidthThin, colorWhite } from '../../styles/designTokens';

const Layout = styled.header`
  position: fixed;
  width: 100%;
  height: 6rem;
  background-color: ${colorWhite};
  border-bottom: ${borderWidthThin} solid ${colorLightGrey};
  top: 0;
`;

export default Layout;
