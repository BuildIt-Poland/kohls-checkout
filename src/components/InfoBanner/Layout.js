import styled from 'styled-components';
import { colorLightBlue, colorWhite, fontSmall, spacingMedium } from '../../styles/designTokens';

const Layout = styled.div`
  display: flex;
  justify-content: center;
  margin-bottom: ${spacingMedium};
  padding: 1rem;
  background-color: ${colorLightBlue};
  color: ${colorWhite};
  text-transform: uppercase;
  font-size: ${fontSmall};
`;

export default Layout;
