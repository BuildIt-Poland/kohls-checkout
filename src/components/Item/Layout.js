import styled from 'styled-components';

import { spacingMedium } from '../../styles/designTokens';

const Layout = styled.div`
  display: grid;
  grid-template-columns: 10rem 1fr;
  grid-gap: ${spacingMedium};
`;

export default Layout;
