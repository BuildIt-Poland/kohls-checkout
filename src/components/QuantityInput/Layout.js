import styled from 'styled-components';

const Layout = styled.div`
  display: grid;
  grid-template-columns: 4rem 12rem 4rem;
  height: 4rem;
`;

export default Layout;
