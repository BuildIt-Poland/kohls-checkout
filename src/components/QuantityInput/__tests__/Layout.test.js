import React from 'react';
import { create } from 'react-test-renderer';

import Layout from '../Layout';

describe('COMPONENT - Item Page', () => {
  it('renders correctly', () => {
    const component = create(<Layout />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
