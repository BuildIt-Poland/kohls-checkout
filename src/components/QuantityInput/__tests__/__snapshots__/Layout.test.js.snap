// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Item Page renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: 4rem 12rem 4rem;
  height: 4rem;
}

@media (max-width:26.25em) {
  .c0 {
    grid-template-columns: 4rem 8rem 4rem;
  }
}

<div
  className="c0"
/>
`;
